google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -372,7 +372,7 @@ def read table, columns, keys: nil, index: nil, limit: nil,
         # @yield [commit] The block for mutating the data.
         # @yieldparam [Google::Cloud::Spanner::Commit] commit The Commit object.
         #
-        # @return [Boolean] Returns `true` if the operation succeeded.
+        # @return [Time] The timestamp at which the operation committed.
```

Comment:
I made these different because when users call `client.insert row` it isn't obvious to them that a single-use transaction is being used. Since the client handles all that for the users I didn't think it made sense to reference a transaction in the return value description.